CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Changes by Version
 ==================
 
+1.17.0 (2019-02-26)
+-------------------
+
+Breaking changes:
+* Removed 'Size' property from components ([#850](https://github.com/jaegertracing/jaeger-operator/pull/850))
```

Comment:
Wondering whether this property can be handled as part of the upgrade procedures? Just in case a user still has it?

Reply:
Good question. I think it's too late now, as it's been deprecated for quite a long time and we just *removed* the property from the CRD. This means that the CRD shouldn't be valid anymore: trying to load a CR with this field should yield an error, as we don't even have this property anymore in the Go code.

Reply:
Possibly something to consider in the future :)

Reply:
For the future, we might consider using something like Kubebuilder's conversion mechanism to migrate from v1alpha1 -> v1 -> v2 -> ...

https://book.kubebuilder.io/multiversion-tutorial/conversion.html